src/sass/extends/_anchor-menu.scss

```diff
@@ -46,10 +46,19 @@
   .nav-link {
     color: $uds-color-base-gray-7;
     padding: $uds-size-spacing-3 0;
-    border-bottom: 1px solid $uds-color-base-gray-3;
     text-align: start;
     font-weight: 400;
 
+    &:not(:last-child) {
```

Comment:
- Feature not implemented in UDS-WordPress. Should be safe to proceed.
- Design of this feature raises questions in Storybook. (Should it really replace the header completely?) 

Recommend waiting to implement until further feedback obtained.

Reply:
Interestingly, I used one of these Anchor Menus in an attempt to improve our blog navigation - using it to display a list of categories as links across the top of `single.php`. It's not actually implemented, and I don't think my solution was the best, anyway. As you say, I don't think we're using this anywhere at the moment.